Marlin/src/inc/Version.h

```diff
@@ -33,7 +33,7 @@
  * vendor name, download location, GitHub account, etc.
  */
 #ifndef DETAILED_BUILD_VERSION
-  #define DETAILED_BUILD_VERSION SHORT_BUILD_VERSION " (knutwurst, Github)" // PATCH: Knutwurst
+  #define DETAILED_BUILD_VERSION "2.0.9.2 (knutwurst, Github)" // PATCH: Knutwurst
```

Comment:
@knutwurst Is there any reason to not change this?

Reply:
Should it not be "Marlin 2.0.9.2 (knutwurst, Github)" 

Do you know if this will be merged into the base?

Reply:
Currently, it's `2.0.x (knutwurst, Github)`.
I don't know 🙂

Reply:
Current output is

Send: M115
Recv: FIRMWARE_NAME:Marlin 2.0.x (knutwurst, Github) (Feb 18 2022 13:26:03) SOURCE_CODE_URL:github.com/knutwurst/Marlin-2-0-x-Anycubic-i3-MEGA-S PROTOCOL_VERSION:1.0 MACHINE_TYPE:Knutwurst's Anycubic Printer EXTRUDER_COUNT:1 UUID:b6153152-86fb-4cd8-bda7-91c6a04c467d

it must pick up the ‘Marlin’ text from somewhere else?  Is it in SHORT_BUILD_VERSION?